docs/network_components.rst

```diff
@@ -0,0 +1,16 @@
+Network Components
+-------------------
+
+.. autofunction:: tensornetwork.contract
+.. autofunction:: tensornetwork.contract_copy_node
+.. autofunction:: tensornetwork.contract_between
+.. autofunction:: tensornetwork.outer_product
+.. autofunction:: tensornetwork.outer_product_final_nodes
+.. autofunction:: tensornetwork.contract_parallel
+.. autofunction:: tensornetwork.flatten_edges
+.. autofunction:: tensornetwork.get_all_nondangling
+.. autofunction:: tensornetwork.get_all_dangling
+.. autofunction:: tensornetwork.flatten_all_edges
+.. autofunction:: tensornetwork.flatten_edges_between
+.. autofunction:: tensornetwork.get_parallel_edges
+.. autofunction:: tensornetwork.get_shared_edges
```

Comment:
I wouldn't separate network components and network operations. There is not difference from the users perspective. Can you combine the two files?

Reply:
Done as well